src/Toolkit.Forms/BasemapGallery/BasemapGallery.Appearance.cs

```diff
@@ -42,22 +42,22 @@ static BasemapGallery()
 
             DefaultGridDataTemplate = new DataTemplate(() =>
             {
-                Grid outerScrimContainer = new Grid();
```

Comment:
All of these changes address the 'new expression can be simplified' build message.

Reply:
We should disable that message instead, as it isn't a requirement for our coding guidelines. I would remove these changes from this PR and we can disable those warnings in a separate PR